components/builder-db/src/migration.rs

```diff
@@ -15,34 +15,55 @@
 use diesel::pg::PgConnection;
 use diesel::query_dsl::RunQueryDsl;
 use diesel::sql_query;
-use error::Result;
 
-pub fn shard_setup(conn: &PgConnection, shard_id: u32) -> Result<()> {
-    debug!("Migrating shard_{:?}", shard_id);
-    sql_query(format!("CREATE SCHEMA IF NOT EXISTS shard_{}", shard_id))
-        .execute(conn)
-        .unwrap();
-    sql_query(format!("SET SEARCH_PATH TO shard_{}", shard_id))
-        .execute(conn)
-        .unwrap();
+use error::{Error, Result};
+use pool::Pool;
+
+pub fn setup_ids(conn: &PgConnection) -> Result<()> {
     // TED: We have to do this here because we rely on the shard id which isn't available in raw sql
```

Comment:
Well, hmm.. now that shard_id is gone, we may as  well move the next_id_v1 now into the main SQL, and kill this function completely.

Reply:
Yep, just add to the migration...

Reply:
Will do

Reply:
Turns out this doesn't work.  Moving this function into the main SQL migrations causes all the other migrations to fail.

Reply:
So I'm going to leave it where it is

Reply:
Ok 👍